src/main/python/pybuilder/pip_utils.py

```diff
@@ -21,14 +21,11 @@
 import sys
 
 from pybuilder.core import Dependency, RequirementsFile
-from pybuilder.pip_common import (Version,
-                                  SpecifierSet,
-                                  search_packages_info,
-                                  pip_working_set_init,
-                                  _pip_disallows_insecure_packages_by_default,
-                                  _pip_supports_constraints,
-                                  pip_version)
 from pybuilder.utils import execute_command, as_list
+# Plugin install_dependencies_plugin can reload pip_common and pip_utils. Do not use from ... import ...
+from pybuilder import pip_common
+Version = pip_common.Version
```

Comment:
I had the same thoughts but we cannot guarantee that we will not change reload modules into future. In this case we should use the same logic for all modules without exception and remove named import from pip_utils (as I did into last changes).